### PR TITLE
Prepare ColumnsMapper

### PR DESCRIPTION
**Pull request by michaljurecko**
Solving: https://keboola.atlassian.net/browse/COM-264

Changes:
- Create `ColumnsMapper`, mapping not yet implemeted.
- Simplified iterating over CSV rows.

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -6,12 +6,12 @@
          convertNoticesToExceptions="true"
          convertWarningsToExceptions="true"
          processIsolation="false"
-         stopOnFailure="true"
+         stopOnFailure="false"
```
Comment (ondrajodas) [inline phpunit.xml.dist:9]: 👍

**Comment by michaljurecko** [comment]
@ondrajodas mozes sa prosim na to pozriet? chcem poslat finalny PR a rebasnut to nad to.

**Comment by ondrajodas** [comment]
jj sry hned jdu na to

**Review by ondrajodas — APPROVED** [review]
ok

**Pull request merged**